dev/conductor/core/lib/src/proto/conductor_state.proto

```diff
@@ -11,16 +11,16 @@ message Remote {
 enum ReleasePhase {
   // Release was started with `conductor start` and repositories cloned.
   APPLY_ENGINE_CHERRYPICKS = 0;
+
+  // Verify engine CI is green before opening framework PR.
   CODESIGN_ENGINE_BINARIES = 1;
```

Comment:
Should we just rename this? Although we persist this proto information to disk, it's really only intended to be read from disk while you're in the middle of a release.

Reply:
Done! Renamed to VERIFY_ENGINE_CI